.github/codecov.yml

```diff
@@ -1,10 +1,15 @@
 # codecov used to be able to find this anywhere, now we have to manually
 # tell it where to look
 comment: false
+ignore:
+  - doc/
+  - galleries/
+  - "test_*.py"
```

Comment:
The coverage on the test files is probably the most important thing to measure.

Reply:
you can not run our complete test suite on a single machine due to the platform specific tests/code and optional dependencies.  These checks are critical to make sure we do not accidentally drop coverage do to changes in CI.

Put another way, tests test our code, but what tests the tests?  Verifying that your tests actually run is about the only thing we have that covers that need.

@dopplershift has also taken this position in the past.